lib/roo_on_rails/checks/environment.rb

```diff
@@ -6,7 +6,6 @@
 require 'roo_on_rails/checks/sidekiq/settings'
 require 'roo_on_rails/checks/heroku/drains_metrics'
 require 'roo_on_rails/checks/documentation/playbook'
-require 'roo_on_rails/checks/google_oauth/initializer'
```

Comment:
Yeah. I'm trying to sheperd a bunch of PRs into master and ran into conflict hell. Fixing this as I write.